src/services/users.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const User = require('../models/users');
+
```

Comment:
use normalization function. There are such fields as `createdAt`, `updatedAt `which is no need to send

Reply:
same for expenses